service/src/main/resources/HTMLTemplate.html

```diff
@@ -1064,7 +1070,7 @@ <h3 class="heading heading--two">Respondent
       </dd>
     </div>
     <div class="data-list__row">
-      <dt class="data-list__title">Current address</dt>
+      <dt class="data-list__title">Current Address</dt>
```

Comment:
Is that an expected change? Seems unrelated to migration and not quite right to me?

Reply:
Yes. Because of the new format, Current Address is used for each entry in new respondents. I want our fixture file to work with both the old respondent AND the new respondent. This was impossible without the above change. It standardises the format. In the old way, "Current Address" was for first respondent, "Current address" was used for the following.

Reply:
I understand need to standardise it 👍